lib/beaker/host/cisco.rb

```diff
@@ -21,7 +21,7 @@ def skip_set_env?
     #
     # @return nil
     def scp_post_operations(scp_file_actual, scp_file_target)
-      if self[:platform] =~ /cisco-5/
+      if self[:platform] =~ /cisco-nexus/
```

Comment:
Same as above: "We found that we could not SCP to the directory on the Nexus, we had to specify the target filename.  That is not required by any other platform."